misago/graphql/admin/analytics.py

```diff
@@ -76,7 +76,22 @@ def get_data_for_model(self, model, date_attr):
                 data[date] += 1
 
         values = list(data.values())
+        current = list(reversed(values[: self.span]))
+        previous = list(reversed(values[self.span :]))
+
         return {
-            "current": list(reversed(values[: self.span])),
-            "previous": list(reversed(values[self.span :])),
+            "current": current,
+            "currentCumulative": cumulate_data(current),
+            "previous": previous,
+            "previousCumulative": cumulate_data(previous)
         }
+
+
+def cumulate_data(data_series):
+    data = []
+    for v in data_series:
```

Comment:
Would it be possible to just use `reduce` on this? 
`return list(reduce(lambda x, y: x + y, data_series))`

Reply:
Nope, `reduce` reduces list of items into a single value ;)

Reply:
Ah! Right 👍